src/pages/Thanks/ThanksPage.vue

```diff
@@ -301,46 +302,55 @@ export default {
 	},
 	apollo: {
 		preFetch(config, client, { cookieStore, route }) {
-			const currentRoute = route.value ?? route ?? {};
-			const transactionId = currentRoute.query?.kiva_transaction_id
-				? numeral(currentRoute.query.kiva_transaction_id).value()
-				: null;
-
-			// Check if transactionId is null, resolve the promise if missing
-			if (!transactionId) {
-				logFormatter(
-					'Thanks page preFetch skipped due to missing transaction_id.',
-					'warning',
-				);
-				return Promise.resolve();
-			}
-
-			return client.query({
-				query: thanksPageQuery,
-				variables: {
-					checkoutId: transactionId,
-					visitorId: cookieStore.get('uiv') || null,
+			return Promise.all([
+				client.query({ query: thanksPageQuery }),
+				client.query({ query: experimentAssignmentQuery, variables: { id: 'share_ask_copy' } }),
+			]).then(() => {
+				const currentRoute = route.value ?? route ?? {};
```

Comment:
Do we still want to do this check before running the two queries? It seems like they don't rely on the results.